src/AutoRest.CSharp/Common/Generation/Writers/ClientWriter.cs

```diff
@@ -8,17 +8,31 @@
 using System.Threading.Tasks;
 using AutoRest.CSharp.Generation.Types;
 using AutoRest.CSharp.Generation.Writers;
+using AutoRest.CSharp.Output.Models;
 using AutoRest.CSharp.Output.Models.Requests;
 using AutoRest.CSharp.Output.Models.Shared;
 using Azure;
 using Azure.Core;
+using Azure.Core.Pipeline;
 
 namespace AutoRest.CSharp.Common.Generation.Writers
 {
     internal class ClientWriter
     {
+        protected const string ClientDiagnosticsVariable = "clientDiagnostics";
```

Comment:
It looks like all of the ClientWriter derived types need this, right?

Reply:
Yes, all the derived classes are using these.